spec/components/pretty_text_spec.rb

```diff
@@ -1783,7 +1783,7 @@ def expect_cooked_match(raw, expected_cooked)
     HTML
   end
 
-  it "has a proper data whitlist on div" do
+  it "has a proper data whitelist on div" do
```

Comment:
I understand that this word is being removed in places -- a disadvantage of typos is that they mean words can hide from such fixes. (I ran into a merge conflict which forced me to drop some other change in this family.)